src/generate-names.h

```diff
@@ -26,7 +26,8 @@ struct Module;
 enum NameOpts {
   None = 0,
   AlphaNames = 1 << 0,
-  NoDollar = 1 << 0,
+  NoDollar = 1 << 1,
```

Comment:
(wait how was this working)
(oh I guess the two were always set together and this was essentially "is wasm-decompile")